### PR TITLE
Feature/issue comments

### PR DESCRIPTION
**Pull request by Oksana098**

### CODE REVIEW

File: client/apps/city_issues/views.py [hunk 1]
```diff
@@ -187,3 +187,27 @@ def get_context_data(self, **kwargs):
...
+
+    def get_context_data(self, **kwargs):
+        context = super(CommentIssues, self).get_context_data(**kwargs)
+        context['issue'] = Issues.objects.get(pk=self.kwargs['pk'])
+        return context
+
+    def form_valid(self, form):
+        form = form.save(commit=False)
+        issue = Issues.objects.get(pk=self.kwargs['pk'])
+        user = User.objects.get(pk=self.request.user.id)
```
Comment (KostSvitlana) [inline client/apps/city_issues/views.py:207]: You dont check if the user is authenticated. Server returns error `User matching query does not exist.`

File: client/apps/city_issues/templates/comment_issue.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                        {% endfor %}
+                    </ul>
+                {% endif %}
+                <table class="table">
+                    <tr>
+                        <th>Title</th>
+                        <th>First Name</th>
+                        <th>Last Name</th>
+                        <th>Sex</th>
+                        <th>Status Person</th>
```
Comment (KostSvitlana) [inline client/apps/city_issues/templates/comment_issue.html]: Very strange column names

**Review by KostSvitlana — COMMENTED** [review]
In this pull request changes from another #48

File: client/apps/city_issues/views.py (hunk 1 above)
Comment (Oksana098) [reply client/apps/city_issues/views.py:207]: fixed

Comment (Oksana098) [reply client/apps/city_issues/views.py:207]: fixed add mixin check auth

Comment (Oksana098) [reply client/apps/city_issues/views.py:207]: look into mixins.py

**Comment by Oksana098** [comment]
yea I see this small fail))

**Review by andrewRv — APPROVED** [review]

File: client/apps/city_issues/views.py (hunk 1 above)
Comment (forever-Agriculture) [reply client/apps/city_issues/views.py:207]: mixins approach, cool

**Review by forever-Agriculture — APPROVED** [review]

File: client/apps/city_issues/views.py (hunk 1 above)
Comment (Oksana098) [reply client/apps/city_issues/views.py:207]: I know that I cool)

**Review by KostSvitlana — APPROVED** [review]

**Pull request merged**